netci.groovy

```diff
@@ -273,7 +275,7 @@ def innerLoopNonWindowsOSs = ['Ubuntu', 'OSX', 'FreeBSD', 'CentOS7.1', 'OpenSUSE
             if (isPR) {
                 // Set PR trigger.
                 // Set of OS's that work currently. 
-                if (os in ['Ubuntu', 'OpenSUSE13.2', 'CentOS7.1']) {
+                if (os in ['Ubuntu', 'Debian8.2', 'OpenSUSE13.2', 'CentOS7.1']) {
```

Comment:
For now, leave this out so we don't launch a potentially broken debian build for every PR.